modules/core/core-content/src/main/java/com/enonic/xp/core/impl/project/CreateProjectIssuesAccessListCommand.java

```diff
@@ -0,0 +1,86 @@
+package com.enonic.xp.core.impl.project;
+
+import com.enonic.xp.project.ProjectConstants;
+import com.enonic.xp.project.ProjectRole;
+import com.enonic.xp.security.RoleKeys;
+import com.enonic.xp.security.acl.AccessControlEntry;
+import com.enonic.xp.security.acl.AccessControlList;
+import com.enonic.xp.security.acl.Permission;
+
+public final class CreateProjectIssuesAccessListCommand
+    extends AbstractProjectCommand
+{
+    private CreateProjectIssuesAccessListCommand( final Builder builder )
+    {
+        super( builder );
+    }
+
+    public static Builder create()
+    {
+        return new Builder();
+    }
+
+    public AccessControlList execute()
+    {
+        return createContentRootPermissions();
+    }
+
+    private AccessControlList createContentRootPermissions()
+    {
+        if ( projectName == null || ProjectConstants.DEFAULT_PROJECT_NAME.equals( projectName ) )
+        {
+            return null;
+        }
+
+        return AccessControlList.create().
+            add( AccessControlEntry.create().
+                allowAll().
+                principal( RoleKeys.ADMIN ).
+                build() ).
+            add( AccessControlEntry.create().
+                allowAll().
+                principal( RoleKeys.CONTENT_MANAGER_ADMIN ).
+                build() ).
+            add( AccessControlEntry.create().
+                allow( Permission.READ, Permission.CREATE, Permission.MODIFY, Permission.DELETE ).
+                principal( createRoleKey( ProjectRole.OWNER ) ).
+                build() ).
+            add( AccessControlEntry.create().
+                allow( Permission.READ, Permission.CREATE, Permission.MODIFY, Permission.DELETE ).
+                principal( createRoleKey( ProjectRole.EDITOR ) ).
+                build() ).
+            add( AccessControlEntry.create().
+                allow( Permission.READ, Permission.CREATE, Permission.MODIFY, Permission.DELETE ).
+                principal( createRoleKey( ProjectRole.AUTHOR ) ).
+                build() ).
+            add( AccessControlEntry.create().
+                allow( Permission.READ, Permission.CREATE, Permission.MODIFY, Permission.DELETE ).
+                principal( createRoleKey( ProjectRole.CONTRIBUTOR ) ).
+                build() ).
+            add( AccessControlEntry.create().
+                allow( Permission.READ ).
```

Comment:
https://github.com/enonic/xp/issues/7986#issuecomment-608337641 I need to have this clarified before accepting PR.